lib/wibox/template.lua

```diff
@@ -232,8 +232,10 @@ function template:set_property(property, value, ids)
     widgets = widgets or {target}
 
     for _, widget in ipairs(widgets) do
-        if widget["set_"..property] then
+        if rawget(widget, "set_"..property) then
             widget["set_"..property](widget, value)
+        else
+            widget[property] = value
```

Comment:
For the reviewers' sakes, it would be nice to keep commits separated by purpose and to not mix small, unrelated and easy-to-miss changes into a file rename.

Reply:
Yeah, sorry, I messed up a rebase somewhere.